client/src/Movies/Movie.js

```diff
@@ -18,39 +22,27 @@ const Movie = (props) => {
           console.error(error);
         });
 
-  },[]);
+  },[props.match.params.dataID]);
   
   // Uncomment this only when you have moved on to the stretch goals
-  // const saveMovie = () => {
-  //   const addToSavedList = props.addToSavedList;
-  //   addToSavedList(movie)
-  // }
+  const saveMovie = () => {
+    const addToSavedList = props.addToSavedList;
+    addToSavedList(movie);
+  }
 
   if (!movie) {
     return <div>Loading movie information...</div>;
   }
 
   const { title, director, metascore, stars } = movie;
+  console.log(movie);
+
   return (
     <div className="save-wrapper">
-      <div className="movie-card">
-        <h2>{title}</h2>
-        <div className="movie-director">
-          Director: <em>{director}</em>
-        </div>
-        <div className="movie-metascore">
-          Metascore: <strong>{metascore}</strong>
-        </div>
-        <h3>Actors</h3>
-
-        {stars.map(star => (
-          <div key={star} className="movie-star">
-            {star}
-          </div>
-        ))}
-      </div>
-      <div className="save-button">Save</div>
-    </div>
+    <MovieCard key={movie.id} title={title} director={director} metascore={metascore} stars={stars}/>
+    <div onClick={()=>saveMovie()} className="save-button">Save</div>
+  </div>
```

Comment:
Good job getting the stretch to work! 
You should be proud and happy you are pushing yourself. 

Because you assigned the saveMovie function to be a const value you don't exactly need the anonymous function from line 43 in your onClick handler.

you can just reference: 
```onClick={saveMovie}```